src/components/Navbar/TopNavbar.tsx

```diff
@@ -1,39 +1,119 @@
-import React from "react";
-import { Navbar, Nav } from "react-bootstrap";
+import React, { useState } from "react";
 import "./TopNavbar.scss";
-import { Link } from "react-router-dom";
+import { Link, NavLink } from "react-router-dom";
 import Logo from "../../assets/logo.svg";
+import loggedProfile from "../../assets/man.png";
+import UnknownProfile from "../../assets/profile.svg";
+import { Button } from "react-bootstrap";
 
 function TopNavbar() {
+  const [isLoggedin, setisLoggedin] = useState(() => {
+    return false;
+  });
+
+  let profilepicture;
+
+  const isloginHandler = () => {
+    if (isloggedin === true) {
+      setisLoggedin(false);
+      localStorage.setItem("isLoggedin", "false");
+    } else {
+      localStorage.setItem("isLoggedin", "true");
+      setisLoggedin(true);
+    }
+  };
+  const isloggedin: boolean = isLoggedin;
+  isloggedin === true
+    ? (profilepicture = loggedProfile)
+    : (profilepicture = UnknownProfile);
```

Comment:
Non actionable: this logic could have been written in the src of the profile image itself, but its okay here for now, no need to change.